addons/account/tests/test_reconciliation_matching_rules.py

```diff
@@ -1104,3 +1104,49 @@ def test_tax_tags_inversion_with_reconciliation_model(self):
             [(6, 0, tax.refund_repartition_line_ids[1].tag_ids.ids)],
             'The tags of the second repartition line are not inverted'
         )
+
+    @freeze_time('2020-01-01')
+    def test_inv_matching_with_write_off_and_foreign_currency(self):
+        journal = self.bank_journal.copy({'currency_id': self.currency_data_2['currency'].id})
```

Comment:
if you don't use the original `bank_journal`, can't you just change its currency? (or maybe there are already some journal entry on it?)

Reply:
Indeed, there are already journal entries ;)